Userland/Libraries/LibArgsParser/ArgsParser.cpp

```diff
@@ -385,7 +382,7 @@ void ArgsParser::print_version(FILE* file)
 {
     // FIXME: Allow applications to override version string for --version.
     //        Especially useful for Lagom applications
-    outln(file, Core::Version::read_long_version_string().release_value_but_fixme_should_propagate_errors());
+    outln(file, String::from_utf8("Version 1.0"sv).release_value_but_fixme_should_propagate_errors());
```

Comment:
The long version also contains the commit hash, which is useful information to have accessible in every application. Can we solve this with weak symbols instead, where LibArgsParser has a weak symbol for getting a version string, LibCore overwrites that symbol with a call to read_long_version_string, and DynamicLoader therefore gets this less informative string instead?